distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -231,25 +235,78 @@ type Table
-    remove_columns self (columns = [self.columns.first.name]) (error_on_missing_columns = False) (on_problems = Report_Warning) =
-        new_columns = self.columns_helper.remove_columns selectors=columns error_on_missing_columns=error_on_missing_columns on_problems=on_problems
+    remove_columns :  Vector (Integer | Text | Regex) | Text | Integer -> Regex -> Boolean -> Problem_Behavior -> Table ! No_Output_Columns | Missing_Input_Columns
+    remove_columns self (columns = [self.columns.first.name]) (case_sensitivity=Case_Sensitivity.Default) (error_on_missing_columns:Boolean=False) (on_problems:Problem_Behavior=Report_Warning) =
+        new_columns = self.columns_helper.remove_columns columns case_sensitivity error_on_missing_columns=error_on_missing_columns on_problems=on_problems
         self.updated_columns new_columns
 
+    ## ALIAS select_na
+       ALIAS select_missing_columns
+
+       Select columns which are either all blank or contain blank values. If no
+       rows are present, all columns are considered blank.
+
+       Arguments:
+       - when_any: By default, only columns consisting of all blank cells are
+         selected. If set to `True`, columns with one or more blank values are
+         selected.
+       - treat_nans_as_blank: If `True`, then `Number.nan` is considered as
+         blank.
+
+       ? Blank values
+         Blank values are `Nothing`, `""` and depending on setting `Number.nan`.
+
+       > Example
+         Select completely blank columns from a table.
+
+             table.select_blank_columns
+    select_blank_columns : Boolean -> Boolean -> Table
+    select_blank_columns self (when_any : Boolean = False) (treat_nans_as_blank : Boolean = False) =
+        new_columns = self.columns_helper.select_blank_columns_helper when_any treat_nans_as_blank
+        if new_columns.length == 0 then Error.throw (No_Output_Columns) else
+            self.updated_columns new_columns
+
+    ## ALIAS drop_na
+       ALIAS drop_missing_columns
+
+       Remove columns which are either all blank or contain blank values. If no
+       rows are present, all columns are considered blank.
+
+       Arguments:
+       - when_any: By default, only columns consisting of all blank cells are
+         selected. If set to `True`, columns with one or more blank values are
+         selected.
+       - treat_nans_as_blank: If `True`, then `Number.nan` is considered as
+         blank.
+
+       ? Blank values
+         Blank values are `Nothing`, `""` and depending on setting `Number.nan`.
+
+       > Example
+         Remove completely blank columns from a table.
+
+             table.remove_blank_columns
+    remove_blank_columns : Boolean -> Boolean -> Table
+    remove_blank_columns self (when_any : Boolean = False) (treat_nans_as_blank : Boolean = False) =
+        new_columns = self.columns_helper.select_blank_columns_helper when_any treat_nans_as_blank True
```

Comment:
I'd use named arguments in such cases, makes it much clearer what this boolean flag represents.

```suggestion
        new_columns = self.columns_helper.select_blank_columns_helper when_any treat_nans_as_blank invert_selection=True
```